templates/vue/src/components/modals/UserSettingsModal/AvatarForm.vue

```diff
@@ -0,0 +1,453 @@
+      ],
+      topTypeOptions: [
+        { value: "NoHair", text: "NoHair" },
+        { value: "Eyepatch", text: "Eyepatch" },
+        { value: "Hat", text: "Hat" },
+        { value: "Hijab", text: "Hijab" },
+        { value: "Turban", text: "Turban" },
+        { value: "WinterHat1", text: "WinterHat1" },
+        { value: "WinterHat2", text: "WinterHat2" },
+        { value: "WinterHat3", text: "WinterHat3" },
+        { value: "WinterHat4", text: "WinterHat4" },
+        { value: "LongHairBigHair", text: "LongHairBigHair" },
+        { value: "LongHairBob", text: "LongHairBob" },
+        { value: "LongHairBun", text: "LongHairBun" },
+        { value: "LongHairCurly", text: "LongHairCurly" },
+        { value: "LongHairCurvy", text: "LongHairCurvy" },
+        { value: "LongHairDreads", text: "LongHairDreads" },
+        { value: "LongHairFrida", text: "LongHairFrida" },
+        { value: "LongHairFro", text: "LongHairFro" },
+        { value: "LongHairFroBand", text: "LongHairFroBand" },
+        { value: "LongHairNotTooLong", text: "LongHairNotTooLong" },
+        { value: "LongHairShavedSides", text: "LongHairShavedSides" },
+        { value: "LongHairMiaWallace", text: "LongHairMiaWallace" },
+        { value: "LongHairStraight", text: "LongHairStraight" },
+        { value: "LongHairStraight2", text: "LongHairStraight2" },
+        { value: "LongHairStraightStrand", text: "LongHairStraightStrand" },
+        { value: "ShortHairDreads01", text: "ShortHairDreads01" },
+        { value: "ShortHairDreads02", text: "ShortHairDreads02" },
+        { value: "ShortHairFrizzle", text: "ShortHairFrizzle" },
+        { value: "ShortHairShaggyMullet", text: "ShortHairShaggyMullet" },
+        { value: "ShortHairShortCurly", text: "ShortHairShortCurly" },
+        { value: "ShortHairShortFlat", text: "ShortHairShortFlat" },
+        { value: "ShortHairShortRound", text: "ShortHairShortRound" },
+        { value: "ShortHairShortWaved", text: "ShortHairShortWaved" },
+        { value: "ShortHairSides", text: "ShortHairSides" },
+        { value: "ShortHairTheCaesar", text: "ShortHairTheCaesar" },
+        { value: "ShortHairTheCaesarSidePart", text: "ShortHairTheCaesarSidePart" },
+      ],
+      topColorOptions: [
+        { value: "Black", text: "Black" },
+        { value: "Blue01", text: "Blue01" },
+        { value: "Blue02", text: "Blue02" },
+        { value: "Blue03", text: "Blue03" },
+        { value: "Gray01", text: "Gray01" },
+        { value: "Gray02", text: "Gray02" },
+        { value: "Heather", text: "Heather" },
+        { value: "PastelBlue", text: "PastelBlue" },
+        { value: "PastelGreen", text: "PastelGreen" },
+        { value: "PastelOrange", text: "PastelOrange" },
+        { value: "PastelRed", text: "PastelRed" },
+        { value: "PastelYellow", text: "PastelYellow" },
+        { value: "Pink", text: "Pink" },
+        { value: "Red", text: "Red" },
+        { value: "White", text: "White" },

```

Comment:
Since there is so much data here maybe putting it into a JS file that exports this static data will make it look cleaner.